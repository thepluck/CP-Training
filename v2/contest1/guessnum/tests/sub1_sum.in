10
4
11
7
4
9
4
32
23
4
2
