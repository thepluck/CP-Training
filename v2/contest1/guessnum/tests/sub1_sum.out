6
25
11
6
17
6
208
105
6
3
